src/util/dom.js

```diff
@@ -53,19 +53,19 @@ exports.contains = contains
 
 let accumulatedCSS = "", cssNode = null
 
-function insertCSS(css) {
-  if (cssNode) cssNode.textContent += css
+function insertCSS(pm, css) {
+  if (pm.cssNode) pm.cssNode.textContent += css
   else accumulatedCSS += css
 }
 exports.insertCSS = insertCSS
 
 // This is called when a ProseMirror instance is created, to ensure
 // the CSS is in the DOM.
-function ensureCSSAdded() {
-  if (!cssNode) {
-    cssNode = document.createElement("style")
-    cssNode.textContent = "/* ProseMirror CSS */\n" + accumulatedCSS
-    document.head.insertBefore(cssNode, document.head.firstChild)
+function ensureCSSAdded(pm) {
+  if (!pm.cssNode) {
```

Comment:
skate will rerender the dom if you don't have `skip` property. We always put `skip` on our `pm` wrapper so it won't get skipped. putting the `<style>` elsewhere, it might be removed.